src/k8s/pkg/k8sd/app/hooks_remove.go

```diff
@@ -53,7 +53,7 @@ func (a *App) onPreRemove(ctx context.Context, s state.State, force bool) (rerr
 
 			log.Info("Deleting node from Kubernetes cluster")
 			if err := c.DeleteNode(ctx, s.Name()); err != nil {
-				log.Error(err, "Failed to remove k8s node %q: %w", s.Name(), err)
+				log.Error(err, fmt.Sprintf("Failed to remove k8s node %q: %v", s.Name(), err))
```

Comment:
`log.Error()` already adds the error. and from line 27 we already have the node name as well in the logger. So this should be `log.Error(err, "Failed to remove Kubernetes node")`